src/ContextTypes/Recipe.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace JsonLd\ContextTypes;
+
+class Recipe extends CreativeWork
+{
+    /**
+     * Property structure
+     * reference: https://schema.org/Recipe (alphabetical order)
+     *
+     * @var array
+     */
+    protected $structure = [
+        'name' => null,
+        'prepTime' => null,
```

Comment:
some attributes seems to be related to an intermediate ContextType called 'HowTo'

Thing > Creative Work > HowTo > Recipe

cf. https://schema.org/HowTo

Maybe move theses attributes into this new contextType is better for future improvements / addition.